packages/cf.js/test/client.spec.ts

```diff
@@ -0,0 +1,85 @@
+import { AppInstance } from "../src/app-instance";
+import { Client } from "../src/client";
+import {
+  MessageDataQuery,
+  NodeMessage,
+  NodeMessageType,
+  NodeProvider,
+  QueryType
+} from "../src/structs";
+
+class TestNodeProvider implements NodeProvider {
+  public postedMessages: NodeMessage[] = [];
+  readonly callbacks: ((message: NodeMessage) => void)[] = [];
+
+  public sendMessageToClient(message: NodeMessage) {
```

Comment:
Can it just be `send` instead of `sendMessageToClient`? Also it's not sending it to a client its sending it to a Node right?